src/js/common/components/NavButtons.jsx

```diff
@@ -33,10 +33,10 @@ export default class NavButtons extends React.Component {
   }
   findNeighbor(increment) {
     const { pages, path, data } = this.props;
-    const currentIndex = pages.map(page => page.name).indexOf(path);
     const filtered = pages.filter(page => {
       return page.depends === undefined || _.matches(page.depends)(data);
     });
+    const currentIndex = filtered.map(page => page.name).indexOf(path);
```

Comment:
This was resulting in an error on the last page of the form if you had a hidden page.